src/main/java/networkmanagementsystem/NetworkManagementClient.java

```diff
@@ -5,670 +5,770 @@
+        String ruName;
+        String ip;
+        int carrierId;
+
+        System.out.println(
+                		"#############################################################\n" +
+                        "##               NETWORK MANAGEMENT CLIENT                 ##\n" +
+                        "##                        TEAM A.2                         ##\n" +
+                        "##               ERICSSON OOP: SUMMER 2021                 ##\n" +
+                        "#############################################################\n");
+        System.out.println("Welcome!");
+        Scanner input = new Scanner(System.in);
+        do {
+            option = "-1";
+            try {
+                String menu = "\nPlease enter the number corresponding to the option you would like to choose:\n"
+                        + "0.	Exit Program\n"
+                        + "1.	Commission Radio Unit\n"
+                        + "2.	Decommission Radio Unit\n"
+                        + "3.	Add Radio Unit\n"
+                        + "4.	Remove Radio Unit\n"
+                        + "5.	Setup Radio Unit\n"
+                        + "6.	Release Radio Unit\n"
+                        + "7.	Activate Radio Unit\n"
+                        + "8.	Deactivate Radio Unit\n"
+                        + "9.	Setup Carrier on Radio Unit\n"
+                        + "10.	Modify Carrier on Radio Unit\n"
+                        + "11.	Remove Carrier on Radio Unit\n"
+                        + "12.	Remove all Carriers on Radio Unit\n"
+                        + "13.	Signal scaling on Radio Unit\n"
+                        + "14.	Post activation\n"
+                        + "15.	Perform self diagnostics\n"
+                        + "16.	List network Inventory\n"
+                        + "17.	List RUs details by standard (RAT type)\n"
+                        + "18.	List RUs details by state\n"
+                        + "19.	List RUs details by band\n"
+                        + "20.	List Radio Unit by IP address\n"
+                        + "21.	Set Alarm on Radio Unit\n"
+                        + "22.	List all Network Alarms\n"
+                        + "23.	Acknowledge Alarm on Radio Unit\n";
+
+                System.out.print(menu);
+                System.out.print(INPUT_PROMPT);
+                option = input.next().trim();
+
+                FrequencyBand freqBand;
+                RatType ratType;
+
+                switch (option) {
+                    case "0":
+                        System.out.println("\nGoodbye!");
+                        input.close();
+                        option = "0";
+                        break;
+                    case "1":
+                        ip = getIpAddress(input);
+                        if (ip.equalsIgnoreCase("BACK")) {
+                            break;
+                        }
+                        networkManagementSys.commissionRu(ip);
+						returnToMenu();
```

Comment:
Minor: we could move all of these calls to `returnToMenu()` outside the switch-statement. This would involve exiting directly from the `case "0":` branch, but I think makes it a little more maintainable

Also some weird indentation in file

Reply:
I had tried this initially, but with certain cases when you type back to get out of a selection you would then be prompted to hit enter, even though you just typed in back to get to the menu. It just felt like it broke the flow a bit, but it's a quick change if you want to switch to it.